### ISSUE TITLE
Bug: Lack of contrast for code blocks in light theme

### ISSUE THREAD

**Issue by intrnl** [mdn/yari#5467]
# Summary

There is a lack of background contrast between code block vs the rest of the articles.

## Where did you spot the bug?

Any articles involving code blocks 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog

### Device

- [x] Desktop
- [x] Mobile: Poco M3
- [ ] Tablet: ...
- [ ] Other: ...

### Browser

- [x] Chrome: 101.0.4919.0 (Official Build) dev (64-bit) 
- [x] Firefox: 99.0a1 nightly
- [ ] Safari: (what version?)
- [ ] Other: (what browser / version?)

### Operating system

- [ ] Windows: (what version?)
- [ ] Mac OS: (what version?)
- [x] Linux: Arch Linux, Liquorix Kernel 5.16.12-lqx1-1-lqx

## Screenshots

![image](https://user-images.githubusercontent.com/20620901/156757521-8320326a-552c-4521-b2ca-70df2070a001.png)

There was clearly an attempt on trying to provide a background for these code blocks, but it's not enough. The background is still too light, failing to provide a contrast with the page background.

**Comment by intrnl** [comment]
Here's how it had looked before.

![image](https://user-images.githubusercontent.com/20620901/156758122-c702aa2c-4fdd-4362-bf0f-0634003709e9.png)

![image](https://user-images.githubusercontent.com/20620901/156758246-66b3d470-3740-44e2-9bc9-5dbb0826c7d9.png)

![image](https://user-images.githubusercontent.com/20620901/156758264-1d53962b-23a4-4f3c-af5b-8feab294733e.png)

**Comment by e3dio** [comment]
Many agree on this, I added my example fix here https://github.com/mdn/yari/issues/5389 note that I have the `code` background color slightly darker than the `pre` block color because the `code` is usually smaller inline section and needs more contrast to see it, the larger `pre` blocks need less because it is larger area easier to see, using the same color it looks too dark

```css
body:not(.dark) #content code {
    background: hsl(240deg 10% 95%);
}
body:not(.dark) #content pre{
    background: hsl(240deg 10% 96%);
}
```

**Comment by mirunacurtean** [comment]
@intrnl This looks to be resolved, to me. Please update if there are other criteria not met here, otherwise we can close this issue. 
![image](https://user-images.githubusercontent.com/53306802/172821332-3f5e1156-b60f-4ebf-a23d-888fe4792387.png)

**Comment by intrnl** [comment]
Sorry, kinda forgot about this issue. The new light theme code blocks are pretty good!
